### DISCUSSION TITLE
Show "frequency" (?) calculation

### DISCUSSION THREAD

**Discussion by mirenbz** [iSoron/uhabits#816] [Feature Requests]
First, sorry if this has been dealt with previously, I haven't been able to find anything (maybe due to inaccurate wording, hence the quotation marks in the title).

I use Loop to track some specific habits that do not have a set goal, and specially in those cases I would like to get some info in the form of "you have done this n times per week/month/ whatever on average" (for the whole period or maybe enabling the option to set certain periods of time like "the last month/year" etc.). Think of taking a medication for example, that you take only when you need it, and want to know how often that is. Would this be possible?

Thanks!

**Comment by herrbrixter** [comment]
Related to #1379
